src/main/java/org/junitpioneer/jupiter/json/JacksonNode.java

```diff
@@ -39,7 +39,7 @@ public boolean isArray() {
 
 	@Override
 	public Stream<Node> elements() {
-		return StreamSupport.stream(node.spliterator(), false).map(node -> new JacksonNode(objectMapper, node));
+		return StreamSupport.stream(node.spliterator(), false).map(n -> new JacksonNode(objectMapper, n));
```

Comment:
Hmmm, is this really an improvement? `element` maybe?

Reply:
👍